++ ls caseresult/ -ltr
++ wc -l
+ filename=case4
+ mkdir caseresult/case4
+ RESULT_DIR=/home/hadoop/git/physical_design/caseresult/case4
+ TEN=10
+ SLEEP_TIME=300
+ YCSB_DIR=/home/hadoop/ycsb-0.1.4
+ number_of_operations=5000000
+ number_of_threads=20
+ number_of_records=5000000
+ output_file_suffix=.dat
+ experiment_list_file=/home/hadoop/git/physical_design/case4.list
+ clean=false
+ create=false
+ load=false
+ random_read=false
+ scan=false
+ all=false
+ for i in '$@'
+ '[' all == all ']'
+ all=true
+ let counter=1
+ cd /home/hadoop/ycsb-0.1.4
++ pwd
+ echo /home/hadoop/ycsb-0.1.4
/home/hadoop/ycsb-0.1.4
+ read line
+ true
+ echo 'droping usertable...'
droping usertable...
+ hbase shell /home/hadoop/git/physical_design/delete_table
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/hadoop/hbase-0.98.3-hadoop1/lib/slf4j-log4j12-1.6.4.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/hadoop/hadoop-1.2.1/lib/slf4j-log4j12-1.4.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
0 row(s) in 33.8840 seconds

0 row(s) in 0.8030 seconds

0 row(s) in 0.0810 seconds

+ echo 'sleeping for 300 ...'
sleeping for 300 ...
+ sleep 300
+ true
+ echo 'creating usertable...'
creating usertable...
+ create_statement='create '\''usertable'\'', { NAME=>'\''family'\'', DATA_BLOCK_ENCODING=>'\''PREFIX'\''}, SPLITS => ['\''user1'\'', '\''user2'\'', '\''user3'\'', '\''user4'\'', '\''user5'\'', '\''user6'\'', '\''user7'\'', '\''user8'\'', '\''user9'\'']'
+ echo create ''\''usertable'\'',' '{' 'NAME=>'\''family'\'',' 'DATA_BLOCK_ENCODING=>'\''PREFIX'\''},' SPLITS '=>' '['\''user1'\'',' ''\''user2'\'',' ''\''user3'\'',' ''\''user4'\'',' ''\''user5'\'',' ''\''user6'\'',' ''\''user7'\'',' ''\''user8'\'',' ''\''user9'\'']'
+ echo 'exit '
+ create_table /home/hadoop/git/physical_design/caseresult/case4/create_1.script
+ echo 'creating table...'
creating table...
+ echo /home/hadoop/git/physical_design/caseresult/case4/create_1.script
/home/hadoop/git/physical_design/caseresult/case4/create_1.script
+ hbase shell /home/hadoop/git/physical_design/caseresult/case4/create_1.script
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/home/hadoop/hbase-0.98.3-hadoop1/lib/slf4j-log4j12-1.6.4.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/hadoop/hadoop-1.2.1/lib/slf4j-log4j12-1.4.3.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
0 row(s) in 1.8240 seconds

+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo 'loading data...'
loading data...
+ echo 'starting time: '
starting time: 
++ date +%s
+ echo 1404343783
1404343783
++ echo /home/hadoop/git/physical_design/caseresult/case4/load_1.dat
+ ./ld 5000000 20 /home/hadoop/git/physical_design/caseresult/case4/load_1.dat
Loading workload...
Starting test.
2014-07-02 16:29:43:771 0 sec: 0 operations; 
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/02 16:29:44 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x46f91a48, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/02 16:29:44 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x46f91a48 connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/02 16:29:44 INFO zookeeper.ClientCnxn: Opening socket connection to server slave1/9.1.143.59:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/02 16:29:44 INFO zookeeper.ClientCnxn: Socket connection established to slave1/9.1.143.59:2181, initiating session
14/07/02 16:29:44 INFO zookeeper.ClientCnxn: Session establishment complete on server slave1/9.1.143.59:2181, sessionid = 0x146f9641277000f, negotiated timeout = 90000
2014-07-02 16:29:53:773 10 sec: 278988 operations; 27893.22 current ops/sec; [INSERT AverageLatency(us)=417.37] 
2014-07-02 16:30:03:774 20 sec: 278988 operations; 0 current ops/sec;  
2014-07-02 16:30:13:775 30 sec: 278988 operations; 0 current ops/sec;  
2014-07-02 16:30:23:775 40 sec: 278988 operations; 0 current ops/sec;  
2014-07-02 16:30:33:778 50 sec: 279042 operations; 5.4 current ops/sec; [INSERT AverageLatency(us)=746558.11] 
2014-07-02 16:30:43:782 60 sec: 365829 operations; 8677.83 current ops/sec; [INSERT AverageLatency(us)=597.43] 
14/07/02 16:30:46 INFO client.AsyncProcess: #6, table=usertable, attempt=10/35 failed 2466 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=710, waitTime=2138, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10069 ms, replay 2466 ops.
14/07/02 16:30:46 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 2483 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=710, waitTime=2138, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10023 ms, replay 2483 ops.
14/07/02 16:30:46 INFO client.AsyncProcess: #8, table=usertable, attempt=10/35 failed 2397 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=710, waitTime=2138, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10055 ms, replay 2397 ops.
14/07/02 16:30:46 INFO client.AsyncProcess: #3, table=usertable, attempt=10/35 failed 2460 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=710, waitTime=2138, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10052 ms, replay 2460 ops.
14/07/02 16:30:46 INFO client.AsyncProcess: #17, table=usertable, attempt=10/35 failed 2467 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=710, waitTime=2138, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10025 ms, replay 2467 ops.
14/07/02 16:30:46 INFO client.AsyncProcess: #5, table=usertable, attempt=10/35 failed 2391 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=710, waitTime=2138, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10072 ms, replay 2391 ops.
14/07/02 16:30:46 INFO client.AsyncProcess: #13, table=usertable, attempt=10/35 failed 2427 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=710, waitTime=2138, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10062 ms, replay 2427 ops.
14/07/02 16:30:46 INFO client.AsyncProcess: #1, table=usertable, attempt=10/35 failed 2399 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=710, waitTime=2138, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10050 ms, replay 2399 ops.
14/07/02 16:30:46 INFO client.AsyncProcess: #7, table=usertable, attempt=10/35 failed 2442 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=710, waitTime=2138, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10022 ms, replay 2442 ops.
14/07/02 16:30:46 INFO client.AsyncProcess: #4, table=usertable, attempt=10/35 failed 2440 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=710, waitTime=2138, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10046 ms, replay 2440 ops.
14/07/02 16:30:49 INFO client.AsyncProcess: #15, table=usertable, attempt=10/35 failed 2441 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=767, waitTime=2178, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 10094 ms, replay 2441 ops.
14/07/02 16:30:49 INFO client.AsyncProcess: #12, table=usertable, attempt=10/35 failed 1883 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=767, waitTime=2178, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 10015 ms, replay 1883 ops.
14/07/02 16:30:49 INFO client.AsyncProcess: #15, table=usertable, attempt=10/35 failed 2412 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=767, waitTime=2178, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 10010 ms, replay 2412 ops.
14/07/02 16:30:49 INFO client.AsyncProcess: #15, table=usertable, attempt=10/35 failed 2436 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=767, waitTime=2178, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 10099 ms, replay 2436 ops.
14/07/02 16:30:49 INFO client.AsyncProcess: #16, table=usertable, attempt=10/35 failed 2465 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=767, waitTime=2178, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 10017 ms, replay 2465 ops.
14/07/02 16:30:49 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 2406 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=767, waitTime=2178, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 10017 ms, replay 2406 ops.
14/07/02 16:30:49 INFO client.AsyncProcess: #17, table=usertable, attempt=10/35 failed 1448 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=767, waitTime=2178, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 10070 ms, replay 1448 ops.
14/07/02 16:30:49 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 2434 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=767, waitTime=2178, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 10058 ms, replay 2434 ops.
14/07/02 16:30:50 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 2440 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=779, waitTime=3027, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 10089 ms, replay 2440 ops.
14/07/02 16:30:50 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 1805 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=779, waitTime=3027, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 10053 ms, replay 1805 ops.
2014-07-02 16:30:53:783 70 sec: 430073 operations; 6421.83 current ops/sec; [INSERT AverageLatency(us)=3553.09] 
14/07/02 16:30:59 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014
14/07/02 16:30:59 INFO client.AsyncProcess: #17, table=usertable, attempt=11/35 failed 2467 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=864, waitTime=2739, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10013 ms, replay 2467 ops.
14/07/02 16:30:59 INFO client.AsyncProcess: #7, table=usertable, attempt=11/35 failed 2442 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=864, waitTime=2739, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10049 ms, replay 2442 ops.
14/07/02 16:30:59 INFO client.AsyncProcess: #3, table=usertable, attempt=11/35 failed 2460 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=864, waitTime=2739, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10065 ms, replay 2460 ops.
14/07/02 16:30:59 INFO client.AsyncProcess: #4, table=usertable, attempt=11/35 failed 2440 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=864, waitTime=2739, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10080 ms, replay 2440 ops.
14/07/02 16:30:59 INFO client.AsyncProcess: #8, table=usertable, attempt=11/35 failed 2397 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=864, waitTime=2739, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10044 ms, replay 2397 ops.
14/07/02 16:30:59 INFO client.AsyncProcess: #1, table=usertable, attempt=11/35 failed 2399 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=864, waitTime=2739, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10022 ms, replay 2399 ops.
14/07/02 16:30:59 INFO client.AsyncProcess: #5, table=usertable, attempt=11/35 failed 2391 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=864, waitTime=2739, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10000 ms, replay 2391 ops.
14/07/02 16:30:59 INFO client.AsyncProcess: #6, table=usertable, attempt=11/35 failed 2466 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=864, waitTime=2739, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10042 ms, replay 2466 ops.
14/07/02 16:30:59 INFO client.AsyncProcess: #13, table=usertable, attempt=11/35 failed 2427 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=864, waitTime=2739, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 10013 ms, replay 2427 ops.
14/07/02 16:31:01 INFO client.AsyncProcess: #12, table=usertable, attempt=11/35 failed 1883 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=906, waitTime=2238, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 10026 ms, replay 1883 ops.
14/07/02 16:31:01 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 failed 2406 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=906, waitTime=2238, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 10046 ms, replay 2406 ops.
14/07/02 16:31:01 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 2434 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=906, waitTime=2238, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 10043 ms, replay 2434 ops.
14/07/02 16:31:01 INFO client.AsyncProcess: #17, table=usertable, attempt=11/35 failed 1448 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=906, waitTime=2238, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 10069 ms, replay 1448 ops.
14/07/02 16:31:01 INFO client.AsyncProcess: #15, table=usertable, attempt=11/35 failed 2412 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=906, waitTime=2238, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 10019 ms, replay 2412 ops.
14/07/02 16:31:01 INFO client.AsyncProcess: #16, table=usertable, attempt=11/35 failed 2465 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=906, waitTime=2238, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 10043 ms, replay 2465 ops.
14/07/02 16:31:01 INFO client.AsyncProcess: #15, table=usertable, attempt=11/35 failed 2436 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=906, waitTime=2238, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 10046 ms, replay 2436 ops.
14/07/02 16:31:01 INFO client.AsyncProcess: #15, table=usertable, attempt=11/35 failed 2441 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=906, waitTime=2238, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 10035 ms, replay 2441 ops.
14/07/02 16:31:02 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 failed 2440 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=919, waitTime=2326, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 10080 ms, replay 2440 ops.
14/07/02 16:31:02 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 1805 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=927, waitTime=2021, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 10074 ms, replay 1805 ops.
2014-07-02 16:31:03:784 80 sec: 434343 operations; 426.96 current ops/sec; [INSERT AverageLatency(us)=18475.79] 
14/07/02 16:31:10 INFO client.AsyncProcess: #13, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014
14/07/02 16:31:10 INFO client.AsyncProcess: #4, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014
14/07/02 16:31:10 INFO client.AsyncProcess: #1, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014
14/07/02 16:31:10 INFO client.AsyncProcess: #3, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014
14/07/02 16:31:11 INFO client.AsyncProcess: #5, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014
14/07/02 16:31:11 INFO client.AsyncProcess: #7, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014
14/07/02 16:31:11 INFO client.AsyncProcess: #6, table=usertable, attempt=12/35 failed 2466 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1087, waitTime=2080, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 20048 ms, replay 2466 ops.
14/07/02 16:31:11 INFO client.AsyncProcess: #17, table=usertable, attempt=12/35 failed 2467 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1087, waitTime=2080, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 20017 ms, replay 2467 ops.
14/07/02 16:31:11 INFO client.AsyncProcess: #8, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014
2014-07-02 16:31:13:785 90 sec: 649587 operations; 21522.25 current ops/sec; [INSERT AverageLatency(us)=3268.58] 
14/07/02 16:31:14 INFO client.AsyncProcess: #12, table=usertable, attempt=12/35 failed 1883 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1150, waitTime=2598, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20139 ms, replay 1883 ops.
14/07/02 16:31:14 INFO client.AsyncProcess: #15, table=usertable, attempt=12/35 failed 2412 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1150, waitTime=2598, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20015 ms, replay 2412 ops.
14/07/02 16:31:14 INFO client.AsyncProcess: #16, table=usertable, attempt=12/35 failed 2465 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1150, waitTime=2598, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20064 ms, replay 2465 ops.
14/07/02 16:31:14 INFO client.AsyncProcess: #15, table=usertable, attempt=12/35 failed 2441 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1150, waitTime=2598, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20150 ms, replay 2441 ops.
14/07/02 16:31:14 INFO client.AsyncProcess: #15, table=usertable, attempt=12/35 failed 2436 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1150, waitTime=2598, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20071 ms, replay 2436 ops.
14/07/02 16:31:14 INFO client.AsyncProcess: #10, table=usertable, attempt=12/35 failed 2406 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1150, waitTime=2598, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20159 ms, replay 2406 ops.
14/07/02 16:31:14 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 2434 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1150, waitTime=2598, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20114 ms, replay 2434 ops.
14/07/02 16:31:14 INFO client.AsyncProcess: #17, table=usertable, attempt=12/35 failed 1448 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1150, waitTime=2598, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20156 ms, replay 1448 ops.
14/07/02 16:31:15 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 failed 1805 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1169, waitTime=3013, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20126 ms, replay 1805 ops.
14/07/02 16:31:15 INFO client.AsyncProcess: #2, table=usertable, attempt=12/35 failed 2440 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1169, waitTime=3013, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20116 ms, replay 2440 ops.
2014-07-02 16:31:23:790 100 sec: 687702 operations; 3809.98 current ops/sec; [INSERT AverageLatency(us)=767.27] 
14/07/02 16:31:33 INFO client.AsyncProcess: #6, table=usertable, attempt=13/35 failed 2466 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1488, waitTime=2149, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 20150 ms, replay 2466 ops.
14/07/02 16:31:33 INFO client.AsyncProcess: #17, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014
2014-07-02 16:31:33:791 110 sec: 697306 operations; 960.21 current ops/sec; [INSERT AverageLatency(us)=12172.69] 
14/07/02 16:31:36 INFO client.AsyncProcess: #16, table=usertable, attempt=13/35 failed 2465 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1543, waitTime=2313, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20060 ms, replay 2465 ops.
14/07/02 16:31:36 INFO client.AsyncProcess: #15, table=usertable, attempt=13/35 failed 2441 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1543, waitTime=2313, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20044 ms, replay 2441 ops.
14/07/02 16:31:36 INFO client.AsyncProcess: #17, table=usertable, attempt=13/35 failed 1448 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1543, waitTime=2313, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20195 ms, replay 1448 ops.
14/07/02 16:31:36 INFO client.AsyncProcess: #10, table=usertable, attempt=13/35 failed 2406 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1543, waitTime=2313, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20116 ms, replay 2406 ops.
14/07/02 16:31:36 INFO client.AsyncProcess: #15, table=usertable, attempt=13/35 failed 2436 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1543, waitTime=2313, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20121 ms, replay 2436 ops.
14/07/02 16:31:36 INFO client.AsyncProcess: #15, table=usertable, attempt=13/35 failed 2412 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1543, waitTime=2313, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20111 ms, replay 2412 ops.
14/07/02 16:31:36 INFO client.AsyncProcess: #12, table=usertable, attempt=13/35 failed 1883 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1543, waitTime=2313, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20071 ms, replay 1883 ops.
14/07/02 16:31:36 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 failed 2434 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=1543, waitTime=2313, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20007 ms, replay 2434 ops.
14/07/02 16:31:37 INFO client.AsyncProcess: #2, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014
14/07/02 16:31:37 INFO client.AsyncProcess: #9, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014
2014-07-02 16:31:43:792 120 sec: 864199 operations; 16687.63 current ops/sec; [INSERT AverageLatency(us)=938.69] 
2014-07-02 16:31:53:792 130 sec: 1029739 operations; 16554 current ops/sec; [INSERT AverageLatency(us)=1231.25] 
14/07/02 16:31:55 INFO client.AsyncProcess: #6, table=usertable, attempt=14/35 failed 2466 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2035, waitTime=2461, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 20123 ms, replay 2466 ops.
14/07/02 16:32:00 INFO client.AsyncProcess: #15, table=usertable, attempt=14/35 failed 2441 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20187 ms, replay 2441 ops.
14/07/02 16:32:00 INFO client.AsyncProcess: #17, table=usertable, attempt=14/35 failed 1448 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20184 ms, replay 1448 ops.
14/07/02 16:32:00 INFO client.AsyncProcess: #15, table=usertable, attempt=14/35 failed 2412 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20077 ms, replay 2412 ops.
14/07/02 16:32:00 INFO client.AsyncProcess: #15, table=usertable, attempt=14/35 failed 2436 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20012 ms, replay 2436 ops.
14/07/02 16:32:00 INFO client.AsyncProcess: #16, table=usertable, attempt=14/35 failed 2465 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20138 ms, replay 2465 ops.
14/07/02 16:32:00 INFO client.AsyncProcess: #2, table=usertable, attempt=14/35 failed 2434 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20111 ms, replay 2434 ops.
14/07/02 16:32:00 INFO client.AsyncProcess: #12, table=usertable, attempt=14/35 failed 1883 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20048 ms, replay 1883 ops.
14/07/02 16:32:00 INFO client.AsyncProcess: #10, table=usertable, attempt=14/35 failed 2406 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20069 ms, replay 2406 ops.
14/07/02 16:32:01 INFO client.AsyncProcess: #5, table=usertable, attempt=10/35 failed 2412 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2147, waitTime=2720, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:11 PDT 2014, retrying after 10099 ms, replay 2412 ops.
14/07/02 16:32:01 INFO client.AsyncProcess: #3, table=usertable, attempt=10/35 failed 2411 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2147, waitTime=2720, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:10 PDT 2014, retrying after 10043 ms, replay 2411 ops.
14/07/02 16:32:02 INFO client.AsyncProcess: #1, table=usertable, attempt=10/35 failed 2502 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2156, waitTime=2957, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:10 PDT 2014, retrying after 10030 ms, replay 2502 ops.
2014-07-02 16:32:03:793 140 sec: 1055501 operations; 2575.94 current ops/sec; [INSERT AverageLatency(us)=1123.08] 
14/07/02 16:32:13 INFO client.AsyncProcess: #5, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:11 PDT 2014
14/07/02 16:32:13 INFO client.AsyncProcess: #3, table=usertable, attempt=11/35 failed 2411 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2263, waitTime=2017, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:10 PDT 2014, retrying after 10009 ms, replay 2411 ops.
14/07/02 16:32:13 INFO client.AsyncProcess: #1, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:10 PDT 2014
2014-07-02 16:32:13:850 150 sec: 1089669 operations; 3397.43 current ops/sec; [INSERT AverageLatency(us)=6068.03] 
14/07/02 16:32:17 INFO client.AsyncProcess: #6, table=usertable, attempt=15/35 failed 2466 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2366, waitTime=3007, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014, retrying after 20008 ms, replay 2466 ops.
14/07/02 16:32:21 INFO client.AsyncProcess: #17, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014
14/07/02 16:32:21 INFO client.AsyncProcess: #15, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014
14/07/02 16:32:22 INFO client.AsyncProcess: #15, table=usertable, attempt=15/35 failed 2412 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2485, waitTime=2469, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20196 ms, replay 2412 ops.
14/07/02 16:32:22 INFO client.AsyncProcess: #10, table=usertable, attempt=15/35 failed 2406 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2485, waitTime=2469, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20095 ms, replay 2406 ops.
14/07/02 16:32:22 INFO client.AsyncProcess: #15, table=usertable, attempt=15/35 failed 2441 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2485, waitTime=2469, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20115 ms, replay 2441 ops.
14/07/02 16:32:22 INFO client.AsyncProcess: #16, table=usertable, attempt=15/35 failed 2465 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2485, waitTime=2469, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20098 ms, replay 2465 ops.
14/07/02 16:32:22 INFO client.AsyncProcess: #2, table=usertable, attempt=15/35 failed 2434 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2485, waitTime=2469, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20017 ms, replay 2434 ops.
14/07/02 16:32:22 INFO client.AsyncProcess: #12, table=usertable, attempt=15/35 failed 1883 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2485, waitTime=2469, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20094 ms, replay 1883 ops.
2014-07-02 16:32:24:056 160 sec: 1220980 operations; 12866.06 current ops/sec; [INSERT AverageLatency(us)=738.82] 
14/07/02 16:32:26 INFO client.AsyncProcess: #3, table=usertable, attempt=12/35 failed 2411 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2532, waitTime=3336, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:10 PDT 2014, retrying after 20063 ms, replay 2411 ops.
2014-07-02 16:32:34:057 170 sec: 1308146 operations; 8715.73 current ops/sec; [INSERT AverageLatency(us)=2145.69] 
14/07/02 16:32:39 INFO client.AsyncProcess: #6, table=usertable, attempt=16/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:46 PDT 2014
2014-07-02 16:32:44:062 180 sec: 1467734 operations; 15950.82 current ops/sec; [INSERT AverageLatency(us)=2207.98] 
14/07/02 16:32:45 INFO client.AsyncProcess: #2, table=usertable, attempt=16/35 failed 2434 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2963, waitTime=2423, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20139 ms, replay 2434 ops.
14/07/02 16:32:45 INFO client.AsyncProcess: #15, table=usertable, attempt=16/35 failed 2441 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2963, waitTime=2423, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20008 ms, replay 2441 ops.
14/07/02 16:32:45 INFO client.AsyncProcess: #16, table=usertable, attempt=16/35 failed 2465 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2963, waitTime=2423, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20050 ms, replay 2465 ops.
14/07/02 16:32:45 INFO client.AsyncProcess: #12, table=usertable, attempt=16/35 failed 1883 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2963, waitTime=2423, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20071 ms, replay 1883 ops.
14/07/02 16:32:45 INFO client.AsyncProcess: #10, table=usertable, attempt=16/35 failed 2406 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2963, waitTime=2423, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014, retrying after 20179 ms, replay 2406 ops.
14/07/02 16:32:45 INFO client.AsyncProcess: #15, table=usertable, attempt=16/35 failed 2412 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=2963, waitTime=2423, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014, retrying after 20049 ms, replay 2412 ops.
14/07/02 16:32:46 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 1420 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3013, waitTime=2000, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:54 PDT 2014, retrying after 10030 ms, replay 1420 ops.
14/07/02 16:32:46 INFO client.AsyncProcess: #20, table=usertable, attempt=10/35 failed 1418 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3012, waitTime=2019, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:54 PDT 2014, retrying after 10009 ms, replay 1418 ops.
14/07/02 16:32:46 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 1059 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3014, waitTime=2024, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:55 PDT 2014, retrying after 10030 ms, replay 1059 ops.
14/07/02 16:32:46 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 2457 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=3014, waitTime=2024, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:53 PDT 2014, retrying after 10077 ms, replay 2457 ops.
14/07/02 16:32:47 INFO client.AsyncProcess: #3, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:10 PDT 2014
2014-07-02 16:32:54:062 190 sec: 1675673 operations; 20793.9 current ops/sec; [INSERT AverageLatency(us)=1022.13] 
14/07/02 16:32:57 INFO client.AsyncProcess: #20, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:54 PDT 2014
14/07/02 16:32:57 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:55 PDT 2014
14/07/02 16:32:57 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:54 PDT 2014
14/07/02 16:32:57 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:31:53 PDT 2014
2014-07-02 16:33:04:063 200 sec: 1826607 operations; 15091.89 current ops/sec; [INSERT AverageLatency(us)=1393.31] 
14/07/02 16:33:05 INFO client.AsyncProcess: #15, table=usertable, attempt=17/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014
14/07/02 16:33:06 INFO client.AsyncProcess: #12, table=usertable, attempt=17/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014
14/07/02 16:33:06 INFO client.AsyncProcess: #16, table=usertable, attempt=17/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014
14/07/02 16:33:06 INFO client.AsyncProcess: #15, table=usertable, attempt=17/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014
14/07/02 16:33:06 INFO client.AsyncProcess: #10, table=usertable, attempt=17/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:51 PDT 2014
14/07/02 16:33:06 INFO client.AsyncProcess: #2, table=usertable, attempt=17/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:29:52 PDT 2014
2014-07-02 16:33:14:064 210 sec: 2057167 operations; 23053.69 current ops/sec; [INSERT AverageLatency(us)=4926.19] 
2014-07-02 16:33:24:065 220 sec: 2159577 operations; 10241 current ops/sec; [INSERT AverageLatency(us)=658.77] 
2014-07-02 16:33:34:065 230 sec: 2159577 operations; 0 current ops/sec;  
2014-07-02 16:33:44:066 240 sec: 2159577 operations; 0 current ops/sec;  
2014-07-02 16:33:54:067 250 sec: 2159577 operations; 0 current ops/sec;  
14/07/02 16:34:00 INFO client.AsyncProcess: #1, table=usertable, attempt=10/35 failed 2436 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4478, waitTime=2325, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:07 PDT 2014, retrying after 10050 ms, replay 2436 ops.
14/07/02 16:34:00 INFO client.AsyncProcess: #2, table=usertable, attempt=10/35 failed 1896 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4478, waitTime=2325, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:07 PDT 2014, retrying after 10039 ms, replay 1896 ops.
2014-07-02 16:34:04:068 260 sec: 2159577 operations; 0 current ops/sec;  
14/07/02 16:34:05 INFO client.AsyncProcess: #4, table=usertable, attempt=10/35 failed 2456 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4516, waitTime=2265, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:09 PDT 2014, retrying after 10048 ms, replay 2456 ops.
14/07/02 16:34:05 INFO client.AsyncProcess: #19, table=usertable, attempt=10/35 failed 2366 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4516, waitTime=2265, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 10070 ms, replay 2366 ops.
14/07/02 16:34:05 INFO client.AsyncProcess: #17, table=usertable, attempt=10/35 failed 2464 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4516, waitTime=2265, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 10069 ms, replay 2464 ops.
14/07/02 16:34:05 INFO client.AsyncProcess: #13, table=usertable, attempt=10/35 failed 2490 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4516, waitTime=2265, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 10083 ms, replay 2490 ops.
14/07/02 16:34:05 INFO client.AsyncProcess: #3, table=usertable, attempt=10/35 failed 2429 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4526, waitTime=2016, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:09 PDT 2014, retrying after 10018 ms, replay 2429 ops.
14/07/02 16:34:05 INFO client.AsyncProcess: #20, table=usertable, attempt=10/35 failed 2392 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4527, waitTime=2002, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 10093 ms, replay 2392 ops.
14/07/02 16:34:11 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 2449 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4624, waitTime=2042, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:13 PDT 2014, retrying after 10020 ms, replay 2449 ops.
14/07/02 16:34:11 INFO client.AsyncProcess: #2, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:07 PDT 2014
14/07/02 16:34:11 INFO client.AsyncProcess: #1, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:07 PDT 2014
2014-07-02 16:34:14:072 270 sec: 2346215 operations; 18656.34 current ops/sec; [INSERT AverageLatency(us)=3632.15] 
14/07/02 16:34:14 INFO client.AsyncProcess: #11, table=usertable, attempt=10/35 failed 2429 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4760, waitTime=2071, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:17 PDT 2014, retrying after 10071 ms, replay 2429 ops.
14/07/02 16:34:14 INFO client.AsyncProcess: #6, table=usertable, attempt=10/35 failed 2471 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4760, waitTime=2071, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014, retrying after 10000 ms, replay 2471 ops.
14/07/02 16:34:14 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 2333 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4760, waitTime=2071, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014, retrying after 10031 ms, replay 2333 ops.
14/07/02 16:34:14 INFO client.AsyncProcess: #8, table=usertable, attempt=10/35 failed 2480 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4760, waitTime=2071, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014, retrying after 10009 ms, replay 2480 ops.
14/07/02 16:34:14 INFO client.AsyncProcess: #16, table=usertable, attempt=10/35 failed 2338 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4760, waitTime=2071, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014, retrying after 10070 ms, replay 2338 ops.
14/07/02 16:34:14 INFO client.AsyncProcess: #12, table=usertable, attempt=10/35 failed 2294 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4760, waitTime=2071, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:17 PDT 2014, retrying after 10007 ms, replay 2294 ops.
14/07/02 16:34:15 INFO client.AsyncProcess: #12, table=usertable, attempt=10/35 failed 2454 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4810, waitTime=2091, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014, retrying after 10042 ms, replay 2454 ops.
14/07/02 16:34:15 INFO client.AsyncProcess: #18, table=usertable, attempt=10/35 failed 2445 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4810, waitTime=2091, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014, retrying after 10066 ms, replay 2445 ops.
14/07/02 16:34:15 INFO client.AsyncProcess: #17, table=usertable, attempt=10/35 failed 2469 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4810, waitTime=2091, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014, retrying after 10079 ms, replay 2469 ops.
14/07/02 16:34:15 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 2417 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4810, waitTime=2091, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:13 PDT 2014, retrying after 10020 ms, replay 2417 ops.
14/07/02 16:34:15 INFO client.AsyncProcess: #6, table=usertable, attempt=10/35 failed 2450 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4810, waitTime=2091, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014, retrying after 10060 ms, replay 2450 ops.
14/07/02 16:34:15 INFO client.AsyncProcess: #5, table=usertable, attempt=10/35 failed 2424 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4818, waitTime=2080, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014, retrying after 10066 ms, replay 2424 ops.
14/07/02 16:34:15 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 2380 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4818, waitTime=2080, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014, retrying after 10021 ms, replay 2380 ops.
14/07/02 16:34:17 INFO client.AsyncProcess: #13, table=usertable, attempt=11/35 failed 2490 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4851, waitTime=2527, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 10056 ms, replay 2490 ops.
14/07/02 16:34:17 INFO client.AsyncProcess: #19, table=usertable, attempt=11/35 failed 2366 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4851, waitTime=2527, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 10055 ms, replay 2366 ops.
14/07/02 16:34:17 INFO client.AsyncProcess: #12, table=usertable, attempt=10/35 failed 1727 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4851, waitTime=2527, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014, retrying after 10045 ms, replay 1727 ops.
14/07/02 16:34:17 INFO client.AsyncProcess: #4, table=usertable, attempt=11/35 failed 2456 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4851, waitTime=2527, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:09 PDT 2014, retrying after 10043 ms, replay 2456 ops.
14/07/02 16:34:17 INFO client.AsyncProcess: #17, table=usertable, attempt=11/35 failed 2464 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4851, waitTime=2527, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 10023 ms, replay 2464 ops.
14/07/02 16:34:17 INFO client.AsyncProcess: #11, table=usertable, attempt=10/35 failed 2387 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4851, waitTime=2527, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:13 PDT 2014, retrying after 10003 ms, replay 2387 ops.
14/07/02 16:34:17 INFO client.AsyncProcess: #3, table=usertable, attempt=11/35 failed 2429 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4874, waitTime=2674, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:09 PDT 2014, retrying after 10096 ms, replay 2429 ops.
14/07/02 16:34:17 INFO client.AsyncProcess: #20, table=usertable, attempt=11/35 failed 2392 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4874, waitTime=2674, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 10025 ms, replay 2392 ops.
14/07/02 16:34:23 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 failed 2449 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=4987, waitTime=2344, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:13 PDT 2014, retrying after 10004 ms, replay 2449 ops.
2014-07-02 16:34:24:072 280 sec: 2368347 operations; 2213.2 current ops/sec; [INSERT AverageLatency(us)=1828.64] 
14/07/02 16:34:27 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 2333 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5018, waitTime=2841, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014, retrying after 10002 ms, replay 2333 ops.
14/07/02 16:34:27 INFO client.AsyncProcess: #16, table=usertable, attempt=11/35 failed 2338 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5018, waitTime=2841, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014, retrying after 10091 ms, replay 2338 ops.
14/07/02 16:34:27 INFO client.AsyncProcess: #11, table=usertable, attempt=11/35 failed 2429 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5018, waitTime=2841, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:17 PDT 2014, retrying after 10080 ms, replay 2429 ops.
14/07/02 16:34:27 INFO client.AsyncProcess: #8, table=usertable, attempt=11/35 failed 2480 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5018, waitTime=2841, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014, retrying after 10048 ms, replay 2480 ops.
14/07/02 16:34:27 INFO client.AsyncProcess: #12, table=usertable, attempt=11/35 failed 2294 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5018, waitTime=2841, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:17 PDT 2014, retrying after 10073 ms, replay 2294 ops.
14/07/02 16:34:27 INFO client.AsyncProcess: #6, table=usertable, attempt=11/35 failed 2471 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5018, waitTime=2841, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014, retrying after 10012 ms, replay 2471 ops.
14/07/02 16:34:30 INFO client.AsyncProcess: #12, table=usertable, attempt=11/35 failed 1727 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014, retrying after 10075 ms, replay 1727 ops.
14/07/02 16:34:30 INFO client.AsyncProcess: #19, table=usertable, attempt=12/35 failed 2366 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 20063 ms, replay 2366 ops.
14/07/02 16:34:30 INFO client.AsyncProcess: #13, table=usertable, attempt=12/35 failed 2490 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 20040 ms, replay 2490 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #4, table=usertable, attempt=12/35 failed 2456 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:09 PDT 2014, retrying after 20101 ms, replay 2456 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #12, table=usertable, attempt=11/35 failed 2454 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5028, waitTime=3136, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014, retrying after 10024 ms, replay 2454 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #18, table=usertable, attempt=11/35 failed 2445 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5028, waitTime=3136, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014, retrying after 10012 ms, replay 2445 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #17, table=usertable, attempt=11/35 failed 2469 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5028, waitTime=3136, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014, retrying after 10011 ms, replay 2469 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #5, table=usertable, attempt=11/35 failed 2424 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5028, waitTime=3136, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014, retrying after 10060 ms, replay 2424 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 2380 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5028, waitTime=3136, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014, retrying after 10078 ms, replay 2380 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #6, table=usertable, attempt=11/35 failed 2450 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5028, waitTime=3136, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014, retrying after 10048 ms, replay 2450 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 failed 2417 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5028, waitTime=3136, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:13 PDT 2014, retrying after 10026 ms, replay 2417 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #11, table=usertable, attempt=11/35 failed 2387 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:13 PDT 2014, retrying after 10067 ms, replay 2387 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #17, table=usertable, attempt=12/35 failed 2464 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34464 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 20025 ms, replay 2464 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #3, table=usertable, attempt=12/35 failed 2429 ops, last exception: java.io.IOException: com.google.protobuf.ServiceException: java.lang.NullPointerException on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:09 PDT 2014, retrying after 20189 ms, replay 2429 ops.
14/07/02 16:34:33 INFO client.AsyncProcess: #20, table=usertable, attempt=12/35 failed 2392 ops, last exception: java.io.IOException: com.google.protobuf.ServiceException: java.lang.NullPointerException on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014, retrying after 20153 ms, replay 2392 ops.
2014-07-02 16:34:34:073 290 sec: 2368347 operations; 0 current ops/sec;  
14/07/02 16:34:35 INFO client.AsyncProcess: #10, table=usertable, attempt=12/35 failed 2449 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34609 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:13 PDT 2014, retrying after 20170 ms, replay 2449 ops.
14/07/02 16:34:37 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 failed 2333 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for write. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34609 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014, retrying after 20158 ms, replay 2333 ops.
14/07/02 16:34:37 INFO client.AsyncProcess: #16, table=usertable, attempt=12/35 failed 2338 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for write. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34609 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014, retrying after 20192 ms, replay 2338 ops.
14/07/02 16:34:37 INFO client.AsyncProcess: #11, table=usertable, attempt=12/35 failed 2429 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for write. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34609 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:17 PDT 2014, retrying after 20057 ms, replay 2429 ops.
14/07/02 16:34:39 INFO client.AsyncProcess: #8, table=usertable, attempt=12/35 failed 2480 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for read. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34610 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014, retrying after 20026 ms, replay 2480 ops.
14/07/02 16:34:39 INFO client.AsyncProcess: #6, table=usertable, attempt=12/35 failed 2471 ops, last exception: java.io.IOException: Unexpected closed connection on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014, retrying after 20119 ms, replay 2471 ops.
14/07/02 16:34:39 INFO client.AsyncProcess: #12, table=usertable, attempt=12/35 failed 2294 ops, last exception: java.io.IOException: Unexpected closed connection on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:17 PDT 2014, retrying after 20034 ms, replay 2294 ops.
14/07/02 16:34:40 INFO client.AsyncProcess: #12, table=usertable, attempt=12/35 failed 1727 ops, last exception: java.net.SocketTimeoutException: Call to slave1/9.1.143.59:60020 failed because java.net.SocketTimeoutException: 2000 millis timeout while waiting for channel to be ready for write. ch : java.nio.channels.SocketChannel[connected local=/9.1.143.58:34610 remote=slave1/9.1.143.59:60020] on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014, retrying after 20075 ms, replay 1727 ops.
2014-07-02 16:34:44:074 300 sec: 2368347 operations; 0 current ops/sec;  
14/07/02 16:34:44 INFO client.AsyncProcess: #17, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014
14/07/02 16:34:44 INFO client.AsyncProcess: #5, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014
14/07/02 16:34:44 INFO client.AsyncProcess: #11, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:13 PDT 2014
14/07/02 16:34:44 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:13 PDT 2014
14/07/02 16:34:44 INFO client.AsyncProcess: #18, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014
14/07/02 16:34:44 INFO client.AsyncProcess: #6, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014
14/07/02 16:34:44 INFO client.AsyncProcess: #12, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014
14/07/02 16:34:44 INFO client.AsyncProcess: #9, table=usertable, attempt=12/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:15 PDT 2014
14/07/02 16:34:51 INFO client.AsyncProcess: #13, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014
14/07/02 16:34:51 INFO client.AsyncProcess: #19, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014
14/07/02 16:34:53 INFO client.AsyncProcess: #17, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014
14/07/02 16:34:53 INFO client.AsyncProcess: #4, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:09 PDT 2014
2014-07-02 16:34:54:075 310 sec: 2677191 operations; 30884.4 current ops/sec; [INSERT AverageLatency(us)=1915.64] 
14/07/02 16:34:54 INFO client.AsyncProcess: #3, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:09 PDT 2014
14/07/02 16:34:54 INFO client.AsyncProcess: #20, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:10 PDT 2014
14/07/02 16:34:56 INFO client.AsyncProcess: #10, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:13 PDT 2014
14/07/02 16:34:59 INFO client.AsyncProcess: #18, table=usertable, attempt=13/35 failed 2333 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5747, waitTime=2148, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014, retrying after 20024 ms, replay 2333 ops.
14/07/02 16:34:59 INFO client.AsyncProcess: #11, table=usertable, attempt=13/35 failed 2429 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=5747, waitTime=2148, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:17 PDT 2014, retrying after 20121 ms, replay 2429 ops.
14/07/02 16:34:59 INFO client.AsyncProcess: #16, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014
14/07/02 16:35:01 INFO client.AsyncProcess: #12, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:17 PDT 2014
14/07/02 16:35:01 INFO client.AsyncProcess: #6, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014
14/07/02 16:35:01 INFO client.AsyncProcess: #8, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:14 PDT 2014
14/07/02 16:35:02 INFO client.AsyncProcess: #12, table=usertable, attempt=13/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014
2014-07-02 16:35:04:075 320 sec: 2964869 operations; 28764.92 current ops/sec; [INSERT AverageLatency(us)=2295.1] 
14/07/02 16:35:11 INFO client.AsyncProcess: #5, waiting for some tasks to finish. Expected max=0, tasksSent=424, tasksDone=423, currentTasksDone=423, retries=92 hasError=false, tableName=usertable
2014-07-02 16:35:14:076 330 sec: 3049390 operations; 8451.25 current ops/sec; [UPDATE AverageLatency(us)=8230406] [INSERT AverageLatency(us)=447.49] [CLEANUP AverageLatency(us)=8230613] 
14/07/02 16:35:22 INFO client.AsyncProcess: #18, table=usertable, attempt=14/35 failed 2333 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=6268, waitTime=2933, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014, retrying after 20105 ms, replay 2333 ops.
14/07/02 16:35:22 INFO client.AsyncProcess: #11, table=usertable, attempt=14/35 failed 2429 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=6268, waitTime=2933, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:17 PDT 2014, retrying after 20105 ms, replay 2429 ops.
2014-07-02 16:35:24:077 340 sec: 3049390 operations; 0 current ops/sec;    
14/07/02 16:35:26 INFO client.AsyncProcess: #5, waiting for some tasks to finish. Expected max=0, tasksSent=427, tasksDone=426, currentTasksDone=426, retries=95 hasError=false, tableName=usertable
2014-07-02 16:35:34:077 350 sec: 3058567 operations; 917.7 current ops/sec;  [INSERT AverageLatency(us)=21576.34]  
14/07/02 16:35:39 INFO client.AsyncProcess: #5, waiting for some tasks to finish. Expected max=0, tasksSent=428, tasksDone=427, currentTasksDone=427, retries=96 hasError=false, tableName=usertable
14/07/02 16:35:43 INFO client.AsyncProcess: #18, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:18 PDT 2014
14/07/02 16:35:44 INFO client.AsyncProcess: #11, table=usertable, attempt=15/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:33:17 PDT 2014
2014-07-02 16:35:44:078 360 sec: 3328919 operations; 27032.5 current ops/sec; [UPDATE AverageLatency(us)=15591200] [INSERT AverageLatency(us)=1468.08] [CLEANUP AverageLatency(us)=15592434.33] 
2014-07-02 16:35:54:119 370 sec: 3609879 operations; 27981.28 current ops/sec;  [INSERT AverageLatency(us)=806.42]  
14/07/02 16:35:58 INFO client.AsyncProcess: #15, table=usertable, attempt=10/35 failed 1755 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=7345, waitTime=2728, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:35:06 PDT 2014, retrying after 10014 ms, replay 1755 ops.
14/07/02 16:35:58 INFO client.AsyncProcess: #9, table=usertable, attempt=10/35 failed 2343 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=7345, waitTime=2728, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:35:03 PDT 2014, retrying after 10034 ms, replay 2343 ops.
14/07/02 16:35:58 INFO client.AsyncProcess: #10, table=usertable, attempt=10/35 failed 2375 ops, last exception: java.io.IOException: Call to slave1/9.1.143.59:60020 failed on local exception: org.apache.hadoop.hbase.ipc.RpcClient$CallTimeoutException: Call id=7345, waitTime=2728, rpcTimetout=2000 on slave1,60020,1404343425077, tracking started Wed Jul 02 16:35:06 PDT 2014, retrying after 10082 ms, replay 2375 ops.
2014-07-02 16:36:04:119 380 sec: 3932527 operations; 32264.8 current ops/sec; [UPDATE AverageLatency(us)=4458565] [INSERT AverageLatency(us)=659.79] [CLEANUP AverageLatency(us)=4458597] 
14/07/02 16:36:09 INFO client.AsyncProcess: #15, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:35:06 PDT 2014
14/07/02 16:36:09 INFO client.AsyncProcess: #9, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:35:03 PDT 2014
14/07/02 16:36:09 INFO client.AsyncProcess: #10, table=usertable, attempt=11/35 SUCCEEDED on slave1,60020,1404343425077, tracking started Wed Jul 02 16:35:06 PDT 2014
2014-07-02 16:36:14:120 390 sec: 4256893 operations; 32433.36 current ops/sec; [UPDATE AverageLatency(us)=2936078.5] [INSERT AverageLatency(us)=678.86] [CLEANUP AverageLatency(us)=2936106] 
2014-07-02 16:36:24:121 400 sec: 4589487 operations; 33256.07 current ops/sec; [UPDATE AverageLatency(us)=4516339] [INSERT AverageLatency(us)=269.42] [CLEANUP AverageLatency(us)=4516370] 
2014-07-02 16:36:34:175 410 sec: 4873897 operations; 28288.24 current ops/sec; [UPDATE AverageLatency(us)=2241068.5] [INSERT AverageLatency(us)=211.74] [CLEANUP AverageLatency(us)=2241095] 
2014-07-02 16:36:41:513 417 sec: 5000000 operations; 17184.93 current ops/sec; [UPDATE AverageLatency(us)=1642922.67] [INSERT AverageLatency(us)=132.44] [CLEANUP AverageLatency(us)=1642947.33] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404344202
1404344202
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo 'random read...'
random read...
+ echo 'starting time: '
starting time: 
++ date +%s
+ echo 1404344212
1404344212
++ echo /home/hadoop/git/physical_design/caseresult/case4/random_read_1.dat
+ ./random_read 5000000 20 /home/hadoop/git/physical_design/caseresult/case4/random_read_1.dat
Loading workload...
Starting test.
2014-07-02 16:36:54:924 0 sec: 0 operations; 
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/02 16:36:55 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x4ef888ea, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/02 16:36:55 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x4ef888ea connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/02 16:36:55 INFO zookeeper.ClientCnxn: Opening socket connection to server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/02 16:36:56 INFO zookeeper.ClientCnxn: Socket connection established to sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, initiating session
14/07/02 16:36:56 INFO zookeeper.ClientCnxn: Session establishment complete on server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, sessionid = 0x46f96417b90009, negotiated timeout = 90000
2014-07-02 16:37:04:926 10 sec: 78912 operations; 7882.53 current ops/sec; [READ AverageLatency(us)=2518.74] 
2014-07-02 16:37:14:928 20 sec: 225298 operations; 14635.67 current ops/sec; [READ AverageLatency(us)=1361.29] 
2014-07-02 16:37:24:930 30 sec: 370490 operations; 14517.75 current ops/sec; [READ AverageLatency(us)=1372.41] 
2014-07-02 16:37:34:931 40 sec: 518881 operations; 14837.62 current ops/sec; [READ AverageLatency(us)=1342.55] 
2014-07-02 16:37:44:931 50 sec: 689424 operations; 17052.59 current ops/sec; [READ AverageLatency(us)=1167.72] 
2014-07-02 16:37:54:932 60 sec: 863519 operations; 17407.76 current ops/sec; [READ AverageLatency(us)=1143.64] 
2014-07-02 16:38:04:932 70 sec: 1033351 operations; 16983.2 current ops/sec; [READ AverageLatency(us)=1167.5] 
2014-07-02 16:38:14:933 80 sec: 1214257 operations; 18088.79 current ops/sec; [READ AverageLatency(us)=1105.05] 
2014-07-02 16:38:24:934 90 sec: 1388804 operations; 17452.95 current ops/sec; [READ AverageLatency(us)=1140.66] 
2014-07-02 16:38:34:935 100 sec: 1562549 operations; 17374.5 current ops/sec; [READ AverageLatency(us)=1146] 
2014-07-02 16:38:44:935 110 sec: 1740284 operations; 17771.72 current ops/sec; [READ AverageLatency(us)=1120.2] 
2014-07-02 16:38:54:936 120 sec: 1919431 operations; 17912.91 current ops/sec; [READ AverageLatency(us)=1111.18] 
2014-07-02 16:39:04:937 130 sec: 2101961 operations; 18251.17 current ops/sec; [READ AverageLatency(us)=1090.55] 
2014-07-02 16:39:14:937 140 sec: 2280156 operations; 17819.5 current ops/sec; [READ AverageLatency(us)=1117.24] 
2014-07-02 16:39:24:938 150 sec: 2457795 operations; 17762.12 current ops/sec; [READ AverageLatency(us)=1120.69] 
2014-07-02 16:39:34:939 160 sec: 2632888 operations; 17507.55 current ops/sec; [READ AverageLatency(us)=1137.23] 
2014-07-02 16:39:44:940 170 sec: 2810196 operations; 17729.03 current ops/sec; [READ AverageLatency(us)=1122.83] 
2014-07-02 16:39:54:940 180 sec: 2993963 operations; 18376.7 current ops/sec; [READ AverageLatency(us)=1083.23] 
2014-07-02 16:40:04:941 190 sec: 3145148 operations; 15116.99 current ops/sec; [READ AverageLatency(us)=1317.64] 
2014-07-02 16:40:14:942 200 sec: 3294225 operations; 14906.21 current ops/sec; [READ AverageLatency(us)=1336.51] 
2014-07-02 16:40:24:942 210 sec: 3443140 operations; 14891.5 current ops/sec; [READ AverageLatency(us)=1337.9] 
2014-07-02 16:40:34:943 220 sec: 3594091 operations; 15093.59 current ops/sec; [READ AverageLatency(us)=1319.79] 
2014-07-02 16:40:44:944 230 sec: 3765390 operations; 17128.19 current ops/sec; [READ AverageLatency(us)=1162.34] 
2014-07-02 16:40:54:944 240 sec: 3940635 operations; 17524.5 current ops/sec; [READ AverageLatency(us)=1136.16] 
2014-07-02 16:41:04:945 250 sec: 4121038 operations; 18038.5 current ops/sec; [READ AverageLatency(us)=1103.53] 
2014-07-02 16:41:14:946 260 sec: 4301314 operations; 18025.8 current ops/sec; [READ AverageLatency(us)=1104.38] 
2014-07-02 16:41:24:947 270 sec: 4487422 operations; 18610.8 current ops/sec; [READ AverageLatency(us)=1043.8] 
2014-07-02 16:41:34:947 280 sec: 4674342 operations; 18690.13 current ops/sec; [READ AverageLatency(us)=1090.71] 
2014-07-02 16:41:44:948 290 sec: 4854260 operations; 17990 current ops/sec; [READ AverageLatency(us)=1106.59] 
2014-07-02 16:41:53:522 298 sec: 5000000 operations; 16997.9 current ops/sec; [UPDATE AverageLatency(us)=12.1] [READ AverageLatency(us)=1120.11] [CLEANUP AverageLatency(us)=140.65] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404344513
1404344513
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloada...
workloada...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404344523
1404344523
++ echo /home/hadoop/git/physical_design/caseresult/case4/workloada_1.dat
+ ./workloada 5000000 20 /home/hadoop/git/physical_design/caseresult/case4/workloada_1.dat
Loading workload...
Starting test.
2014-07-02 16:42:04:364 0 sec: 0 operations; 
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/02 16:42:04 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x6b355469, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/02 16:42:04 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x6b355469 connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/02 16:42:04 INFO zookeeper.ClientCnxn: Opening socket connection to server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/02 16:42:04 INFO zookeeper.ClientCnxn: Socket connection established to sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, initiating session
14/07/02 16:42:04 INFO zookeeper.ClientCnxn: Session establishment complete on server sceplus-vm48.almaden.ibm.com/9.1.143.58:2181, sessionid = 0x46f96417b9000a, negotiated timeout = 90000
2014-07-02 16:42:14:365 10 sec: 38452 operations; 3844.82 current ops/sec; [INSERT AverageLatency(us)=554.13] [READ AverageLatency(us)=3111.71] 
2014-07-02 16:42:24:368 20 sec: 143984 operations; 10551.09 current ops/sec; [INSERT AverageLatency(us)=15.84] [READ AverageLatency(us)=1889.68] 
2014-07-02 16:42:34:368 30 sec: 270700 operations; 12670.33 current ops/sec; [INSERT AverageLatency(us)=14] [READ AverageLatency(us)=1730.86] 
2014-07-02 16:42:44:370 40 sec: 396591 operations; 12586.58 current ops/sec; [INSERT AverageLatency(us)=12.57] [READ AverageLatency(us)=1744.2] 
2014-07-02 16:42:54:370 50 sec: 518965 operations; 12237.4 current ops/sec; [INSERT AverageLatency(us)=14.43] [READ AverageLatency(us)=1797.4] 
2014-07-02 16:43:04:371 60 sec: 677653 operations; 15867.21 current ops/sec; [INSERT AverageLatency(us)=14.23] [READ AverageLatency(us)=1383.98] 
2014-07-02 16:43:14:372 70 sec: 825445 operations; 14777.72 current ops/sec; [INSERT AverageLatency(us)=16.88] [READ AverageLatency(us)=1491.33] 
2014-07-02 16:43:24:372 80 sec: 947323 operations; 12187.8 current ops/sec; [INSERT AverageLatency(us)=45.98] [READ AverageLatency(us)=1807.51] 
2014-07-02 16:43:34:373 90 sec: 1067199 operations; 11986.4 current ops/sec; [INSERT AverageLatency(us)=19.5] [READ AverageLatency(us)=1841.65] 
2014-07-02 16:43:44:373 100 sec: 1202619 operations; 13542 current ops/sec; [INSERT AverageLatency(us)=14.61] [READ AverageLatency(us)=1628.76] 
2014-07-02 16:43:54:374 110 sec: 1355167 operations; 15253.27 current ops/sec; [INSERT AverageLatency(us)=13.16] [READ AverageLatency(us)=1447.16] 
2014-07-02 16:44:04:375 120 sec: 1512329 operations; 15714.63 current ops/sec; [INSERT AverageLatency(us)=13.76] [READ AverageLatency(us)=1403.54] 
2014-07-02 16:44:14:375 130 sec: 1672884 operations; 16055.5 current ops/sec; [INSERT AverageLatency(us)=14.17] [READ AverageLatency(us)=1374.81] 
2014-07-02 16:44:24:376 140 sec: 1845325 operations; 17242.38 current ops/sec; [INSERT AverageLatency(us)=15.23] [READ AverageLatency(us)=1278.32] 
2014-07-02 16:44:34:377 150 sec: 1983849 operations; 13852.4 current ops/sec; [INSERT AverageLatency(us)=24.04] [READ AverageLatency(us)=1594.84] 
2014-07-02 16:44:44:377 160 sec: 2165995 operations; 18212.78 current ops/sec; [INSERT AverageLatency(us)=19.69] [READ AverageLatency(us)=1211.23] 
2014-07-02 16:44:54:378 170 sec: 2340837 operations; 17482.45 current ops/sec; [INSERT AverageLatency(us)=13.97] [READ AverageLatency(us)=1263.91] 
2014-07-02 16:45:04:378 180 sec: 2518689 operations; 17785.2 current ops/sec; [INSERT AverageLatency(us)=14.18] [READ AverageLatency(us)=1239.28] 
2014-07-02 16:45:14:379 190 sec: 2697084 operations; 17837.72 current ops/sec; [INSERT AverageLatency(us)=14.26] [READ AverageLatency(us)=1234.94] 
2014-07-02 16:45:24:380 200 sec: 2844182 operations; 14708.33 current ops/sec; [INSERT AverageLatency(us)=19.09] [READ AverageLatency(us)=1496.34] 
2014-07-02 16:45:34:380 210 sec: 3000282 operations; 15610 current ops/sec; [INSERT AverageLatency(us)=15.77] [READ AverageLatency(us)=1420.31] 
2014-07-02 16:45:44:382 220 sec: 3182070 operations; 18175.16 current ops/sec; [INSERT AverageLatency(us)=14.04] [READ AverageLatency(us)=1214.31] 
2014-07-02 16:45:54:383 230 sec: 3370800 operations; 18873 current ops/sec; [INSERT AverageLatency(us)=13.85] [READ AverageLatency(us)=1140.45] 
2014-07-02 16:46:04:384 240 sec: 3557243 operations; 18640.57 current ops/sec; [INSERT AverageLatency(us)=13.69] [READ AverageLatency(us)=1213.35] 
2014-07-02 16:46:14:385 250 sec: 3725208 operations; 16794.82 current ops/sec; [INSERT AverageLatency(us)=17.38] [READ AverageLatency(us)=1303.71] 
2014-07-02 16:46:24:385 260 sec: 3868906 operations; 14369.8 current ops/sec; [INSERT AverageLatency(us)=25.03] [READ AverageLatency(us)=1544.1] 
2014-07-02 16:46:34:390 270 sec: 4036475 operations; 16748.53 current ops/sec; [INSERT AverageLatency(us)=15.45] [READ AverageLatency(us)=1316.56] 
2014-07-02 16:46:44:391 280 sec: 4215172 operations; 17869.7 current ops/sec; [INSERT AverageLatency(us)=14.79] [READ AverageLatency(us)=1234.9] 
2014-07-02 16:46:54:394 290 sec: 4407951 operations; 19270.19 current ops/sec; [INSERT AverageLatency(us)=14.74] [READ AverageLatency(us)=1142.88] 
2014-07-02 16:47:04:395 300 sec: 4593735 operations; 18576.54 current ops/sec; [INSERT AverageLatency(us)=17.06] [READ AverageLatency(us)=1187.48] 
2014-07-02 16:47:14:395 310 sec: 4738368 operations; 14463.3 current ops/sec; [INSERT AverageLatency(us)=27.21] [READ AverageLatency(us)=1525.94] 
2014-07-02 16:47:24:396 320 sec: 4899442 operations; 16105.79 current ops/sec; [INSERT AverageLatency(us)=15.29] [READ AverageLatency(us)=1371.59] 
2014-07-02 16:47:32:546 328 sec: 5000000 operations; 12338.4 current ops/sec; [UPDATE AverageLatency(us)=141786.9] [INSERT AverageLatency(us)=19.27] [READ AverageLatency(us)=1418.34] [CLEANUP AverageLatency(us)=141959.9] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404344852
1404344852
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloadb...
workloadb...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404344863
1404344863
++ echo /home/hadoop/git/physical_design/caseresult/case4/workloadb_1.dat
+ ./workloadb 5000000 20 /home/hadoop/git/physical_design/caseresult/case4/workloadb_1.dat
Loading workload...
Starting test.
2014-07-02 16:47:43:434 0 sec: 0 operations; 
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/02 16:47:43 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0x69e0b7a2, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/02 16:47:43 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0x69e0b7a2 connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/02 16:47:43 INFO zookeeper.ClientCnxn: Opening socket connection to server slave1/9.1.143.59:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/02 16:47:43 INFO zookeeper.ClientCnxn: Socket connection established to slave1/9.1.143.59:2181, initiating session
14/07/02 16:47:43 INFO zookeeper.ClientCnxn: Session establishment complete on server slave1/9.1.143.59:2181, sessionid = 0x146f96412770010, negotiated timeout = 90000
2014-07-02 16:47:53:436 10 sec: 48195 operations; 4817.09 current ops/sec; [INSERT AverageLatency(us)=598.61] [READ AverageLatency(us)=2886.74] 
2014-07-02 16:48:03:438 20 sec: 214544 operations; 16631.57 current ops/sec; [INSERT AverageLatency(us)=17.54] [READ AverageLatency(us)=2245.85] 
2014-07-02 16:48:13:439 30 sec: 380911 operations; 16635.04 current ops/sec; [INSERT AverageLatency(us)=17.39] [READ AverageLatency(us)=2344.35] 
2014-07-02 16:48:23:440 40 sec: 561188 operations; 18025.9 current ops/sec; [INSERT AverageLatency(us)=30.15] [READ AverageLatency(us)=2149.38] 
2014-07-02 16:48:33:440 50 sec: 690734 operations; 12954.6 current ops/sec; [INSERT AverageLatency(us)=15.98] [READ AverageLatency(us)=3071.26] 
2014-07-02 16:48:43:441 60 sec: 842202 operations; 15145.29 current ops/sec; [INSERT AverageLatency(us)=15.29] [READ AverageLatency(us)=2611.09] 
2014-07-02 16:48:53:442 70 sec: 981759 operations; 13954.3 current ops/sec; [INSERT AverageLatency(us)=17.62] [READ AverageLatency(us)=2829.42] 
2014-07-02 16:49:03:442 80 sec: 1134473 operations; 15271.4 current ops/sec; [INSERT AverageLatency(us)=13.87] [READ AverageLatency(us)=2591.99] 
2014-07-02 16:49:13:443 90 sec: 1305051 operations; 17056.09 current ops/sec; [INSERT AverageLatency(us)=21.75] [READ AverageLatency(us)=2296] 
2014-07-02 16:49:23:444 100 sec: 1473917 operations; 16884.91 current ops/sec; [INSERT AverageLatency(us)=19.1] [READ AverageLatency(us)=2355.43] 
2014-07-02 16:49:33:445 110 sec: 1637323 operations; 16338.97 current ops/sec; [INSERT AverageLatency(us)=34.5] [READ AverageLatency(us)=2390.12] 
2014-07-02 16:49:43:446 120 sec: 1731116 operations; 9378.36 current ops/sec; [INSERT AverageLatency(us)=30.39] [READ AverageLatency(us)=4218.33] 
2014-07-02 16:49:53:448 130 sec: 1883605 operations; 15245.85 current ops/sec; [INSERT AverageLatency(us)=24.24] [READ AverageLatency(us)=2549.84] 
2014-07-02 16:50:03:449 140 sec: 2043442 operations; 15982.1 current ops/sec; [INSERT AverageLatency(us)=17.46] [READ AverageLatency(us)=2484.2] 
2014-07-02 16:50:13:450 150 sec: 2158859 operations; 11540.55 current ops/sec; [INSERT AverageLatency(us)=17.18] [READ AverageLatency(us)=3441.61] 
2014-07-02 16:50:23:451 160 sec: 2282097 operations; 12322.57 current ops/sec; [INSERT AverageLatency(us)=16.47] [READ AverageLatency(us)=3196.23] 
2014-07-02 16:50:33:452 170 sec: 2433180 operations; 15106.79 current ops/sec; [INSERT AverageLatency(us)=15.81] [READ AverageLatency(us)=2629.12] 
2014-07-02 16:50:43:453 180 sec: 2568620 operations; 13542.65 current ops/sec; [INSERT AverageLatency(us)=16.85] [READ AverageLatency(us)=2920.02] 
2014-07-02 16:50:53:453 190 sec: 2715419 operations; 14679.9 current ops/sec; [INSERT AverageLatency(us)=15.94] [READ AverageLatency(us)=2696.38] 
2014-07-02 16:51:03:454 200 sec: 2886112 operations; 17067.59 current ops/sec; [INSERT AverageLatency(us)=16.26] [READ AverageLatency(us)=2314.03] 
2014-07-02 16:51:13:455 210 sec: 3035854 operations; 14972.7 current ops/sec; [INSERT AverageLatency(us)=17] [READ AverageLatency(us)=2657.75] 
2014-07-02 16:51:23:455 220 sec: 3219075 operations; 18322.1 current ops/sec; [INSERT AverageLatency(us)=16.63] [READ AverageLatency(us)=2153.1] 
2014-07-02 16:51:33:456 230 sec: 3392932 operations; 17383.96 current ops/sec; [INSERT AverageLatency(us)=16.59] [READ AverageLatency(us)=2232.59] 
2014-07-02 16:51:43:457 240 sec: 3540179 operations; 14723.23 current ops/sec; [INSERT AverageLatency(us)=17.89] [READ AverageLatency(us)=2735.09] 
2014-07-02 16:51:53:458 250 sec: 3677285 operations; 13709.23 current ops/sec; [INSERT AverageLatency(us)=14.88] [READ AverageLatency(us)=2906.71] 
2014-07-02 16:52:03:459 260 sec: 3750250 operations; 7296.5 current ops/sec; [INSERT AverageLatency(us)=18.23] [READ AverageLatency(us)=3980.06] 
2014-07-02 16:52:13:459 270 sec: 3750250 operations; 0 current ops/sec;   
2014-07-02 16:52:23:460 280 sec: 3750250 operations; 0 current ops/sec;   
2014-07-02 16:52:33:461 290 sec: 3779581 operations; 2932.81 current ops/sec; [INSERT AverageLatency(us)=18.85] [READ AverageLatency(us)=44626.39] 
2014-07-02 16:52:43:462 300 sec: 3857284 operations; 7769.52 current ops/sec; [INSERT AverageLatency(us)=12.91] [READ AverageLatency(us)=5163.76] 
2014-07-02 16:52:53:479 310 sec: 3930473 operations; 7306.48 current ops/sec; [INSERT AverageLatency(us)=14.03] [READ AverageLatency(us)=5472.11] 
2014-07-02 16:53:03:480 320 sec: 3999980 operations; 6950 current ops/sec; [INSERT AverageLatency(us)=15.44] [READ AverageLatency(us)=5747.24] 
2014-07-02 16:53:13:480 330 sec: 4160927 operations; 16094.7 current ops/sec; [INSERT AverageLatency(us)=15.7] [READ AverageLatency(us)=2460.9] 
2014-07-02 16:53:23:482 340 sec: 4328405 operations; 16744.45 current ops/sec; [INSERT AverageLatency(us)=13.21] [READ AverageLatency(us)=2361.59] 
2014-07-02 16:53:33:483 350 sec: 4491538 operations; 16311.67 current ops/sec; [INSERT AverageLatency(us)=17.27] [READ AverageLatency(us)=2427.69] 
2014-07-02 16:53:43:483 360 sec: 4660972 operations; 16943.4 current ops/sec; [INSERT AverageLatency(us)=14.63] [READ AverageLatency(us)=2329.93] 
2014-07-02 16:53:53:484 370 sec: 4825271 operations; 16428.26 current ops/sec; [INSERT AverageLatency(us)=17.85] [READ AverageLatency(us)=2395.25] 
2014-07-02 16:54:03:485 380 sec: 4957733 operations; 13244.88 current ops/sec; [INSERT AverageLatency(us)=19.01] [READ AverageLatency(us)=2956.01] 
2014-07-02 16:54:09:172 385 sec: 5000000 operations; 7432.21 current ops/sec; [UPDATE AverageLatency(us)=209408.7] [INSERT AverageLatency(us)=14.82] [READ AverageLatency(us)=2456.39] [CLEANUP AverageLatency(us)=210047.55] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404345249
1404345249
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ true
+ echo workloadc...
workloadc...
+ echo 'start time: '
start time: 
++ date +%s
+ echo 1404345259
1404345259
++ echo /home/hadoop/git/physical_design/caseresult/case4/workloadc_1.dat
+ ./workloadc 5000000 20 /home/hadoop/git/physical_design/caseresult/case4/workloadc_1.dat
Loading workload...
Starting test.
2014-07-02 16:54:20:556 0 sec: 0 operations; 
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:zookeeper.version=3.4.6-1569965, built on 02/20/2014 09:09 GMT
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:host.name=sceplus-vm48.almaden.ibm.com
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:java.version=1.6.0_31
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:java.vendor=Sun Microsystems Inc.
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:java.home=/usr/lib/jvm/java-6-openjdk-amd64/jre
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:java.class.path=/home/hadoop/ycsb-0.1.4/hbase-binding/lib/hbase-binding-0.1.4.jar:/home/hadoop/ycsb-0.1.4/hbase-binding/conf:/home/hadoop/ycsb-0.1.4/jdbc-binding/conf:/home/hadoop/ycsb-0.1.4/core/lib/core-0.1.4.jar
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:java.library.path=/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64/server:/usr/lib/jvm/java-6-openjdk-amd64/jre/lib/amd64:/usr/lib/jvm/java-6-openjdk-amd64/jre/../lib/amd64:/usr/java/packages/lib/amd64:/usr/lib/jni:/lib:/usr/lib
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:java.io.tmpdir=/tmp
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:java.compiler=<NA>
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:os.name=Linux
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:os.arch=amd64
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:os.version=3.13.0-24-generic
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:user.name=hadoop
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:user.home=/home/hadoop
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Client environment:user.dir=/home/hadoop/ycsb-0.1.4
14/07/02 16:54:20 INFO zookeeper.ZooKeeper: Initiating client connection, connectString=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181 sessionTimeout=90000 watcher=hconnection-0xc2c915f, quorum=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181, baseZNode=/hbase
14/07/02 16:54:21 INFO zookeeper.RecoverableZooKeeper: Process identifier=hconnection-0xc2c915f connecting to ZooKeeper ensemble=sceplus-vm49.almaden.ibm.com:2181,sceplus-vm48.almaden.ibm.com:2181
14/07/02 16:54:21 INFO zookeeper.ClientCnxn: Opening socket connection to server slave1/9.1.143.59:2181. Will not attempt to authenticate using SASL (unknown error)
14/07/02 16:54:21 INFO zookeeper.ClientCnxn: Socket connection established to slave1/9.1.143.59:2181, initiating session
14/07/02 16:54:21 INFO zookeeper.ClientCnxn: Session establishment complete on server slave1/9.1.143.59:2181, sessionid = 0x146f96412770011, negotiated timeout = 90000
2014-07-02 16:54:30:559 10 sec: 171548 operations; 17147.94 current ops/sec; [INSERT AverageLatency(us)=173.51] [READ AverageLatency(us)=6986.49] 
2014-07-02 16:54:40:560 20 sec: 422625 operations; 25105.19 current ops/sec; [INSERT AverageLatency(us)=15.57] [READ AverageLatency(us)=7668.61] 
2014-07-02 16:54:50:561 30 sec: 733903 operations; 31124.69 current ops/sec; [INSERT AverageLatency(us)=22.13] [READ AverageLatency(us)=6033.16] 
2014-07-02 16:55:00:562 40 sec: 974612 operations; 24068.49 current ops/sec; [INSERT AverageLatency(us)=20.54] [READ AverageLatency(us)=8057.88] 
2014-07-02 16:55:10:563 50 sec: 1162161 operations; 18753.02 current ops/sec; [INSERT AverageLatency(us)=21.91] [READ AverageLatency(us)=10526.24] 
2014-07-02 16:55:20:564 60 sec: 1380234 operations; 21805.12 current ops/sec; [INSERT AverageLatency(us)=14.98] [READ AverageLatency(us)=8994.71] 
2014-07-02 16:55:30:565 70 sec: 1587052 operations; 20679.73 current ops/sec; [INSERT AverageLatency(us)=14.36] [READ AverageLatency(us)=9552.83] 
2014-07-02 16:55:40:572 80 sec: 1792665 operations; 20546.92 current ops/sec; [INSERT AverageLatency(us)=17.14] [READ AverageLatency(us)=9450.79] 
2014-07-02 16:55:50:598 90 sec: 1943299 operations; 15024.34 current ops/sec; [INSERT AverageLatency(us)=14.42] [READ AverageLatency(us)=12584.6] 
2014-07-02 16:56:00:599 100 sec: 2096672 operations; 15335.77 current ops/sec; [INSERT AverageLatency(us)=17.68] [READ AverageLatency(us)=13305.36] 
2014-07-02 16:56:10:599 110 sec: 2297466 operations; 20079.4 current ops/sec; [INSERT AverageLatency(us)=12.88] [READ AverageLatency(us)=9722] 
2014-07-02 16:56:20:624 120 sec: 2465895 operations; 16800.9 current ops/sec; [INSERT AverageLatency(us)=17.74] [READ AverageLatency(us)=11573.38] 
2014-07-02 16:56:30:632 130 sec: 2594471 operations; 12847.32 current ops/sec; [INSERT AverageLatency(us)=14.26] [READ AverageLatency(us)=14564.71] 
2014-07-02 16:56:40:632 140 sec: 2748131 operations; 15366 current ops/sec; [INSERT AverageLatency(us)=13.43] [READ AverageLatency(us)=13631.51] 
2014-07-02 16:56:50:633 150 sec: 2873953 operations; 12580.94 current ops/sec; [INSERT AverageLatency(us)=14.81] [READ AverageLatency(us)=15878.04] 
2014-07-02 16:57:00:633 160 sec: 3034072 operations; 16011.9 current ops/sec; [INSERT AverageLatency(us)=17.84] [READ AverageLatency(us)=12310.76] 
2014-07-02 16:57:10:634 170 sec: 3200765 operations; 16667.63 current ops/sec; [INSERT AverageLatency(us)=12.58] [READ AverageLatency(us)=11944.32] 
2014-07-02 16:57:20:635 180 sec: 3378910 operations; 17812.72 current ops/sec; [INSERT AverageLatency(us)=13.41] [READ AverageLatency(us)=11051.63] 
2014-07-02 16:57:30:636 190 sec: 3562095 operations; 18318.5 current ops/sec; [INSERT AverageLatency(us)=14.66] [READ AverageLatency(us)=10713.7] 
2014-07-02 16:57:40:637 200 sec: 3691268 operations; 12914.72 current ops/sec; [INSERT AverageLatency(us)=13.18] [READ AverageLatency(us)=15709.75] 
2014-07-02 16:57:50:638 210 sec: 3866764 operations; 17549.6 current ops/sec; [INSERT AverageLatency(us)=13.03] [READ AverageLatency(us)=11147.59] 
2014-07-02 16:58:00:638 220 sec: 4063356 operations; 19657.23 current ops/sec; [INSERT AverageLatency(us)=12.19] [READ AverageLatency(us)=10125.6] 
2014-07-02 16:58:10:639 230 sec: 4230730 operations; 16735.73 current ops/sec; [INSERT AverageLatency(us)=15.82] [READ AverageLatency(us)=11656.14] 
2014-07-02 16:58:20:654 240 sec: 4386532 operations; 15556.86 current ops/sec; [INSERT AverageLatency(us)=14.04] [READ AverageLatency(us)=12860.74] 
2014-07-02 16:58:30:655 250 sec: 4515978 operations; 12943.31 current ops/sec; [INSERT AverageLatency(us)=12.1] [READ AverageLatency(us)=15047.78] 
2014-07-02 16:58:40:655 260 sec: 4672031 operations; 15605.3 current ops/sec; [INSERT AverageLatency(us)=11.94] [READ AverageLatency(us)=12738.32] 
2014-07-02 16:58:50:680 270 sec: 4845073 operations; 17261.05 current ops/sec; [INSERT AverageLatency(us)=13.34] [READ AverageLatency(us)=10963.32] 
2014-07-02 16:59:00:681 280 sec: 4995036 operations; 14994.8 current ops/sec; [UPDATE AverageLatency(us)=404675.09] [INSERT AverageLatency(us)=12.19] [READ AverageLatency(us)=11640.74] [CLEANUP AverageLatency(us)=406999.09] 
2014-07-02 16:59:02:509 281 sec: 5000000 operations; 2715.54 current ops/sec; [UPDATE AverageLatency(us)=314283.33] [INSERT AverageLatency(us)=11.97] [READ AverageLatency(us)=11659.22] [CLEANUP AverageLatency(us)=314302.22] 
+ echo 'end time:'
end time:
++ date +%s
+ echo 1404345543
1404345543
+ echo 'sleeping for 10 sec ...'
sleeping for 10 sec ...
+ sleep 10
+ let counter=1+1
+ read line
+ set +x
